plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -530,195 +558,205 @@ But in our case this is okay, because:
 
 -- | Call 'dischargeCekValue' over the received 'CekVal' and feed the resulting 'Term' to
 -- 'throwErrorWithCause' as the cause of the failure.
-throwErrorDischarged
-  :: ThrowableBuiltins uni fun
-  => EvaluationError (MachineError fun) CekUserError
-  -> CekValue uni fun ann
-  -> CekM uni fun s x
+throwErrorDischarged ::
+  ThrowableBuiltins uni fun =>
+  EvaluationError (MachineError fun) CekUserError ->
+  CekValue uni fun ann ->
+  CekM uni fun s x
```

Comment:
I prefer having arrows in the beginning of the line. I think there was some option in fourmolu to force this.